assets/stylesheet.css

```diff
@@ -25,6 +25,10 @@ h1 {
 	margin-bottom: 0.375em;
 }
 
+.page-content {
```

Comment:
I noticed that the content was very narrow for only the Register page, and realized it was only wider on the other pages due to their longer `h1` text. I suggest this as a consistent width across pages and welcome thoughts/feedback!

Reply:
This might be a question for Yinka. I think more typically we would set a more specific size for a content section, or maybe vary it based on breakpoints. Feels like more of a design than an engineering question in the end, but especially at smaller sizes this feels too narrow, personally.